### ISSUE TITLE
Add an argument for setting the working directory

### ISSUE THREAD

**Issue by jp7677** [jspm/jspm-cli#2091]
Hi there,

Thanks for all you work on jspm. Once you got the hang of it, it is really cool! 
I have a small wish though: A lot of node tools for web development support a working directory as argument, e.g.

```
npm install --prefix ui
ui/node_modules/.bin/typings install --cwd ui
ui/node_modules/.bin/gulp --cwd ui
```
It would be cool if jspm had a similar switch.

**Comment by guybedford** [comment]
Thanks for the feedback, yes something like this would be useful certainly.

**Comment by jp7677** [comment]
Implemented on master and 0.17.

**Comment by jp7677** [comment]
PS: Looking forward to a new release ;).
